edx_proctoring/api.py

```diff
@@ -1646,8 +1648,10 @@ def get_attempt_status_summary(user_id, course_id, content_id):
     attempt = get_exam_attempt(exam['id'], user_id)
     if attempt:
         status = attempt['status']
-    elif not exam['is_practice_exam'] \
-            and credit_state and has_due_date_passed(credit_state.get('course_end_date', None)):
+    elif (
+        not exam['is_practice_exam'] and credit_state and  # pylint: disable=C0330
```

Comment:
I've added `pylint disable` here because pylint warning looks like a checker's bug. 

One of the complained issue's thread https://github.com/PyCQA/pylint/issues/741

Reply:
What I would do here to avoid the `pylint diable` is to just use more variables to capture the conditions we want to test out. I think use more lines of code and variables make the code easier to read.
```
attempt = get_exam_attempt(exam['id'], user_id)
graded_exam = not exam['is_practice_exam']
due_date_is_passed = credit_state and has_due_date_passed(credit_state.get('course_end_date', None))

if attempt:
    status = attempt['status']
elif graded_exam and due_date_is_passed:
    status = ProctoredExamStudentAttemptStatus.expired
...
```